public/enhanced-table-vis.js

```diff
@@ -19,117 +19,126 @@
 
 import { i18n } from '@kbn/i18n';
 import { AggGroupNames } from '../../../src/plugins/data/public';
-import { Schemas } from '../../../src/legacy/core_plugins/vis_default_editor/public';
-import { prepareJson, prepareString } from '../../../src/legacy/core_plugins/visualizations/public/np_ready/public/legacy/build_pipeline';
+import { Schemas } from '../../../src/plugins/vis_default_editor/public';
+import { prepareJson, prepareString } from '../../../src/plugins/visualizations/public/legacy/build_pipeline';
 
 import tableVisTemplate from './enhanced-table-vis.html';
-import { EnhancedTableVisualizationController } from './vis_controller';
+import { getEnhancedTableVisualizationController } from './vis_controller';
 import { enhancedTableRequestHandler } from './data_load/enhanced-table-request-handler';
 import { enhancedTableResponseHandler } from './data_load/enhanced-table-response-handler';
 import { EnhancedTableOptions } from './components/enhanced_table_vis_options';
 
+const toExpression = (vis) => {
```

Comment:
Same request here, could you replace the line by:
`function toExpression(vis) {`